ts/src/index.ts

```diff
@@ -18,6 +18,7 @@ import * as utils from "./utils";
 import { Program } from "./program";
 import { Address } from "./program/common";
 import { Event } from "./program/event";
+export * from "./error";
```

Comment:
Nit: I dont have push access but this should be below immediately next to the other export.